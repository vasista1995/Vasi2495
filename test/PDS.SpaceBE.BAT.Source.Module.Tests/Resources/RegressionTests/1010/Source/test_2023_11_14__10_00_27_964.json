{
  "Device": "S0946S",
  "ParameterClass": "PPAP",
  "CFComment": "FOL",
  "Segment": "A2PL",
  "F56Parameter": "N",
  "FourDReport": "*",
  "SpaceRawValues": [
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 235960053,
      "Seqnr": 1,
      "Value": 163.87,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 1,
      "ViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE",
      "ViolationList": "18",
      "PrimaryViolation": "18",
      "PrimaryViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE"
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 235960053,
      "Seqnr": 6,
      "Value": 161.15,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 1,
      "ViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE",
      "ViolationList": "18",
      "PrimaryViolation": "18",
      "PrimaryViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE"
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 235960053,
      "Seqnr": 5,
      "Value": 161.23,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 1,
      "ViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE",
      "ViolationList": "18",
      "PrimaryViolation": "18",
      "PrimaryViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE"
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 235960053,
      "Seqnr": 2,
      "Value": 161.28,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 1,
      "ViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE",
      "ViolationList": "18",
      "PrimaryViolation": "18",
      "PrimaryViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE"
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 235960053,
      "Seqnr": 3,
      "Value": 164.94,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 1,
      "ViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE",
      "ViolationList": "18",
      "PrimaryViolation": "18",
      "PrimaryViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE"
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 235960053,
      "Seqnr": 4,
      "Value": 158.61,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 1,
      "ViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE",
      "ViolationList": "18",
      "PrimaryViolation": "18",
      "PrimaryViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE"
    }
  ],
  "UniqueSourceKey": "BAT:BE:RU310249B05:BE1I:FOL_Autovision:A2PLBallShearTop:61701:235960053:3",
  "PKey": "BAT:BE:RU310249B05:BE1I:FOL_Autovision:A2PLBallShearTop:61701:235960053:3",
  "ParameterName": "A2PLBallShearTop",
  "ParameterUnit": "g",
  "SampleDate": "2023-10-01T00:48:09Z",
  "SampleDateUtc": "2023-09-30T17:48:09Z",
  "SampleSize": 6,
  "Sigma": "2,25057029809483",
  "Mean": "161,846666666667",
  "Median": "161,255",
  "Min": "158,61",
  "Max": "164,94",
  "Q1": "161,17",
  "Q3": "163,2225",
  "LdsID": "1010",
  "IsCalcParam": 0,
  "MeanCntrlLow": 129.69136005536,
  "MeanCntrlTarget": 153.02,
  "MeanCntrlHigh": 176.34863994464,
  "RangeCntrlLow": 7.36840205373988,
  "RangeCntrlTarget": 11.9903361056735,
  "RangeCntrlHigh": 38.646690416496,
  "SigmaCntrlLow": 2.78234550501781,
  "SpecLow": 60.0,
  "SpecTarget": 100.0,
  "CtrlMeanHighEnabled": "Y",
  "CtrlMeanLowEnabled": "Y",
  "CtrlMeanTargetEnabled": "Y",
  "CtrlLowEnabled": "N",
  "CtrlHighEnabled": "N",
  "CtrlRangeLowEnabled": "Y",
  "CtrlRangeHighEnabled": "Y",
  "CtrlSigmaLowEnabled": "Y",
  "CtrlSigmaCenterEnabled": "N",
  "CtrlRangeCenterEnabled": "Y",
  "CtrlCenterEnabled": "N",
  "CtrlSigmaHighEnabled": "N",
  "SpecLowEnabled": "Y",
  "SpecHighEnabled": "N",
  "CreatedDate": "2023-10-01T00:48:09Z",
  "CreatedDateUtc": "2023-09-30T17:48:09Z",
  "UpdatedDate": "2023-11-14T15:02:56Z",
  "UpdatedDateUtc": "2023-11-14T08:02:56Z",
  "RvStoreFlag": "Y",
  "Operation": "FOL_Autovision",
  "Facility": "BE1I",
  "ChannelId": "61701",
  "CkcId": "3",
  "ChannelName": "Ball_ShearForce/CU/50um/PDS20/A2CU/S0946S",
  "ChannelDescr": "A2PLBallShearTop\\PDS20\\A2CU\\50",
  "SpaceInstanceName": "BE",
  "Sitekey": "BAT",
  "CalcCreated": "06/26/2023 08:22:48",
  "ILdsID": 0,
  "AcceptFlag": "Y",
  "UpdatedBy": "iscw8cc2082mwr",
  "SpecTargetOrigin": "V",
  "ViolCount": 1,
  "ExtMaMV": 151.544375,
  "ExtEwmaMV": 154.052575471442,
  "ExtMsMV": 16.2799551255183,
  "OutOfOrder": "N",
  "EwmaS": 4.6225534370096,
  "EwmaR": 11.4691715785568,
  "Calc2ID": 2315009,
  "LimitEnable": "YYYYNNYNYYYYYYYYYYYNNNNNNNNNYNNNNNNNN",
  "ExtEwmaMVLCL": 141.233667467736,
  "ExtEwmaMVCenter": 153.02,
  "ExtEwmaMVUCL": 164.806332532264,
  "ExtMaMVLCL": 144.685804341116,
  "ExtMaMVCenter": 153.02,
  "ExtMaMVUCL": 161.354195658884,
  "EwmaSLCL": 2.0181882374845,
  "EwmaSCenter": 5.2963181807713,
  "EwmaSUCL": 8.5744481240581,
  "EwmaRLCL": 4.52194841798305,
  "EwmaRCenter": 13.9648,
  "EwmaRUCL": 23.4076515820169,
  "ExtMSMVLCL": 0.0152568325790999,
  "ExtMSMVCenter": 7.19466418466009,
  "ExtMSMVUCL": 28.9012446554143,
  "ClOrigin": 0,
  "MVLal": 0.0,
  "MVUal": 0.0,
  "RawLal": 111.332529935211,
  "RawUal": 194.707470064789,
  "SigmaLal": 0.869982693009067,
  "SigmaUal": 20.5248923987803,
  "RangeLal": 2.74646800180622,
  "RangeUal": 65.3030447273185,
  "ExtSpecLower": 0.0,
  "ExtSpecUpper": 0.0,
  "ExtSpecLimEnable": "NN",
  "CalculationStrategy": "ND",
  "ChannelType": "N",
  "ChState": "Online"
}