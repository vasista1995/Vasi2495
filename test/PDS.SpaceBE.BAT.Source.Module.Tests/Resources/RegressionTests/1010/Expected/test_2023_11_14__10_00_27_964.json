{
  "IdSource": "SPACEACT2:BAT:BE:RU310249B05:BE1I:FOL_Autovision:A2PLBallShearTop:61701:235960053:3",
  "Data_flat_Metadata": {
    "Device": "S0946S",
    "ParameterClass": "PPAP",
    "CFComment": "FOL",
    "Segment": "A2PL",
    "F56Parameter": "N",
    "FourDReport": "*",
    "ParameterName": "A2PLBallShearTop",
    "ParameterUnit": "g",
    "ParameterOper": "FOL_Autovision",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "ChannelId": "61701",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-09-30T17:48:09Z",
    "CreatedTimestamp": "2023-10-01T00:48:09Z",
    "UpdatedTimestamp": "2023-11-14T15:02:56Z",
    "UpdatedTimestampUtc": "2023-11-14T08:02:56Z",
    "ExportedTimestampUtc": "2023-10-01T00:48:09Z",
    "ExportedTimestamp": "2023-10-01T00:48:09Z",
    "SampleTimestampUtc": "2023-09-30T17:48:09Z",
    "SampleTimestamp": "2023-10-01T00:48:09Z",
    "SourceDataLevel": "C",
    "ChannelName": "Ball_ShearForce/CU/50um/PDS20/A2CU/S0946S",
    "ChannelDescr": "A2PLBallShearTop\\PDS20\\A2CU\\50",
    "RvStoreFlag": "Y",
    "CalcCreated": "06/26/2023 08:22:48",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": "iscw8cc2082mwr",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 2315009,
    "ViolCount": 1,
    "CalculationStrategy": "ND",
    "LdsID": "1010",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "3",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "N",
        "SpecLowEnabled": "Y",
        "SpecLow": 60.0,
        "SpecTarget": 100.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYYYNNYNYYYYYYYYYYYNNNNNNNNNYNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "161,846666666667",
        "Median": "161,255",
        "Sigma": "2,25057029809483",
        "Min": "158,61",
        "Max": "164,94",
        "Q1": "161,17",
        "Q3": "163,2225"
      },
      "ControlLimits": {
        "ctrl_mean_low": 129.69136005536,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 153.02,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 176.34863994464,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "ctrl_sigma_low": 2.78234550501781,
        "CtrlSigmaLowEnabled": "Y",
        "CtrlSigmaHighEnabled": "N",
        "ctrl_range_low": 7.36840205373988,
        "CtrlRangeLowEnabled": "Y",
        "ctrl_range_target": 11.9903361056735,
        "ctrl_range_high": 38.646690416496,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 151.544375,
        "ExtEwmaMV": 154.052575471442,
        "ExtMsMV": 16.2799551255183,
        "EwmaS": 4.6225534370096,
        "EwmaR": 11.4691715785568,
        "ExtEwmaMVLCL": 141.233667467736,
        "ExtEwmaMVCenter": 153.02,
        "ExtEwmaMVUCL": 164.806332532264,
        "ExtMaMVLCL": 144.685804341116,
        "ExtMaMVCenter": 153.02,
        "ExtMaMVUCL": 161.354195658884,
        "EwmaSLCL": 2.0181882374845,
        "EwmaSCenter": 5.2963181807713,
        "EwmaSUCL": 8.5744481240581,
        "EwmaRLCL": 4.52194841798305,
        "EwmaRCenter": 13.9648,
        "EwmaRUCL": 23.4076515820169,
        "ExtMSMVLCL": 0.0152568325790999,
        "ExtMSMVCenter": 7.19466418466009,
        "ExtMSMVUCL": 28.9012446554143,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 111.332529935211,
        "RawUal": 194.707470064789,
        "SigmaLal": 0.869982693009067,
        "SigmaUal": 20.5248923987803,
        "RangeLal": 2.74646800180622,
        "RangeUal": 65.3030447273185
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 235960053,
        "Seqnr": 1,
        "Value": 163.87,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "18",
        "PrimaryViolation": "18",
        "PrimaryViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE"
      },
      {
        "SampleId": 235960053,
        "Seqnr": 6,
        "Value": 161.15,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "18",
        "PrimaryViolation": "18",
        "PrimaryViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE"
      },
      {
        "SampleId": 235960053,
        "Seqnr": 5,
        "Value": 161.23,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "18",
        "PrimaryViolation": "18",
        "PrimaryViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE"
      },
      {
        "SampleId": 235960053,
        "Seqnr": 2,
        "Value": 161.28,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "18",
        "PrimaryViolation": "18",
        "PrimaryViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE"
      },
      {
        "SampleId": 235960053,
        "Seqnr": 3,
        "Value": 164.94,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "18",
        "PrimaryViolation": "18",
        "PrimaryViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE"
      },
      {
        "SampleId": 235960053,
        "Seqnr": 4,
        "Value": 158.61,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "18",
        "PrimaryViolation": "18",
        "PrimaryViolationComments": "RC   : A2P BALLSHEAR OUT OF LCL RANGE\nCA   : UNDER MONITORING BY QE/UPE"
      }
    ],
    "SampleSize": 6
  },
  "ProductionAction": {
    "Id": "SPACEACT1:BAT:BE:BE1I:FOL_Autovision:A2PLBallShearTop:61701",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "BAT:BE:RU310249B05:BE1I:FOL_Autovision:A2PLBallShearTop:61701:235960053:3"
  }
}